style1.css

```diff
@@ -0,0 +1,98 @@
+*{
+  padding: 0;
+  margin: 0;
+}
+body, html{
+  overflow-x: hidden;
+}
+nav{
+  margin: 0 auto;
+  max-width: 1100px;
+  width: 100%;
+}
+#checkbox-menu,
+.toggle{
+  display:none;
+}
+.menu{
+  padding: 0;
+  margin: 0;
+  height: 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  list-style-type: none;
+}
+.menu li a{
+  color: #000;
+  text-decoration: none;
+  align-self: center;
+  font-size: 13px;
+  padding: 10px 15px;
+  transition: background .2s linear;  
+}
+.menu li a:hover{
+  background: #ffcc01;
+  border-radius: 3px;
+}
+.menu li a.logo{
+  margin-right: 240px;
+  font-size: 25px;
+}
+.menu li a.contacts{
+  margin-right: 100px;
+}
+
+@media screen and (max-width: 765px) {
+  .toggle{
+    clear: both;
+    display: block;
+    text-align: center;
+    font-size: 18px;
+    line-height: 40px;
+    width: 100%;
+    color: #000;
+    background: #ffcc01;
+    transition: all .1s linear;
+  }
+  .toggle:hover{
+    background: #d3a900;
+  }
+  #checkbox-menu:checked + label .touch li {
+    opacity: 1;
+    visibility: visible;
+    transition: all .7s linear;
+  }
+  #checkbox-menu:checked + label .touch {
+    height: 350px;
+  }
+  .menu {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    height: 0;
+    transition: height .3s linear;
+  }
+  .menu li {
+    display: flex;
+    align-self: center;
+    width: 95%;
+    opacity: 0;
+    visibility: hidden;
+  }
+  .menu li a{
+    width: 95%;
+    text-align: center;
+    align-self: center;
+    align-content: center;
+  }
+  .menu li a.logo{
+    margin-right: 0;
+    font-size: 21px;
+  }
+  .menu li a.contacts{
+    margin-right: 0;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/media_adaptive/pullRequest?prid=5498104)